yktrace/Cargo.toml

```diff
@@ -11,6 +11,7 @@ fallible-iterator = "0.2.0"
 fxhash = "0.2.1"
 gimli = "0.23.0"
 hwtracer = { git = "https://github.com/softdevteam/hwtracer" }
+intervaltree = "0.2.6"
```

Comment:
In general, I suggest just saying `0.2` (cargo will treat `0.2.6` as a *minimum* not a mandatory requirement, so it suggests a level of precision that doesn't really exist in practise).

Reply:
I stopped doing that ever since I discovered that `cargo update` inserts a full version number. This way we get less noise from running `cargo update`.

Reply:
Inserts a version number where? I'm not sure how this lessens noise but I'm prepared to be taught!

Reply:
I was wrong.

I thought that if I have say "0.2" in the Cargo.toml, that `cargo update` would expand it to e.g. `0.2.1`. Nope.

Fixed in f0643c9